actionmailer/test/helper.rb

```diff
@@ -0,0 +1,4 @@
+if ENV['COVERAGE']

```

Comment:
Can we have this live in `abstract_unit`, to avoid spreading requires everywhere?